domain/urgentWishForChildren.unit.ts

```diff
@@ -35,20 +35,26 @@ describe("Urgent Wish for Children", () => {
 
             const mutations = UrgentWishForChildren.execute(game, player.id);
 
+            const actionSpaceDwarf = getActionSpaceNewDwarf(mutations);
+            const playerDwarf = getPlayerNewDwarf(mutations, player);
+
+            expect(actionSpaceDwarf).to.deep.equal(playerDwarf);
```

Comment:
@yangmenghan [Detail] You could also write it as `expect(getActionSpaceNewDwarf(mutations)).to.deep.equal(getPlayerNewDwarf(mutations, player));` if you prefer

But that's debatable, I let you see